prowler/providers/aws/services/ec2/ec2_find_secrets_launch_template/ec2_find_secrets_launch_template.metadata.json

```diff
@@ -0,0 +1,34 @@
+{
+  "Provider": "aws",
+  "CheckID": "ec2_find_secrets_launch_template",
+  "CheckTitle": "Find secrets in EC2 Launch Template",
+  "CheckType": [
+    "IAM"
+  ],
+  "ServiceName": "ec2",
+  "SubServiceName": "",
+  "ResourceIdTemplate": "arn:partition:service:region:account-id:",
+  "Severity": "critical",
+  "ResourceType": "Other",
```

Comment:
```suggestion
  "ResourceType": "AwsEc2LaunchTemplate",
```

Reply:
According to https://docs.aws.amazon.com/securityhub/latest/userguide/asff-resourcedetails-awsec2.html#asff-resourcedetails-awsec2launchtemplate